blueprint/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Beta: Update the presence of a Zoom user based upon an inbound interaction
+title: "Beta: Update the presence of a Zoom user based upon an inbound interaction"
```

Comment:
@carnellj-genesys @DannaShirley be sure to make sure the frontmatter is valid. This broke the devcenter build.